### ISSUE TITLE
API documentation for AbstractConsumer

### ISSUE THREAD

**Issue by philipaconrad** [veegee/amqpy#35]
In the [online docs][amqpy-docs], API documentation exists for almost all of the fundamental stuff *except* the `AbstractConsumer` class, which is subclassed in the usage examples.

After looking in the code, it seems the class already has docstrings in place. Is there a reason for why its docs are missing? (like API instability, or somesuch?)

   [amqpy-docs]: http://amqpy.readthedocs.io/en/latest/

**Comment by veegee** [comment]
Yes, the library is not at 1.0.0 and the undocumented stuff should be considered unstable. I haven't found a nice and clean way to create consumers short of just doing a simple function callback.

**Comment by philipaconrad** [comment]
Thanks for the quick reply!

Your reply answers my question completely, but I'm not sure if closing the issue is the right thing to do, since the docs are not up, and won't be until the creation process for consumers stabilizes.

Maybe tag this issue `enhancement`, and leave it alone for now?
